test/unit/evaluator_test.rb

```diff
@@ -0,0 +1,84 @@
+require 'test_helper'
+require 'support/boolean_or_bust_expression_resolver'
+
+class EvaluatorTest < Test::Unit::TestCase
+  include Sparkql
+
+  def test_simple
+    assert sample('Test Eq true')
+    assert !sample('Test Eq false')
+    assert sample("Test Eq 'Drop'")
+  end
+
+  def test_conjunction
+    assert sample('Test Eq true And Test Eq true')
+    assert !sample('Test Eq false And Test Eq true')
+    assert !sample('Test Eq false And Test Eq false')
+    # Ors
+    assert sample("Test Eq true Or Test Eq true")
+    assert sample("Test Eq true Or Test Eq false")
+    assert sample("Test Eq false Or Test Eq true")
+    assert !sample("Test Eq false Or Test Eq false")
```

Comment:
Whoops, I closed this out before seeing the last couple comments. I think this is a reasonable addition to be confident things are working properly.